admin/templates/pre_reg/draft_detail.html

```diff
@@ -12,15 +12,43 @@
 {% render_bundle 'vendor' %}
 <link href="{% static 'css/registrations.css' %}" rel="stylesheet" type="text/css" />
 <script type="application/javascript">
-   window.contextVars = window.contextVars || {};
-   window.contextVars.currentUser = {
-     id: -1,
-     fullname: 'Prereg Admin'
-   };
-   window.contextVars.draft = {{ draft | jsonify | safe }};
-   window.contextVars.urls = {
-     updateDraft: '{% url 'pre_reg:update_draft' draft.pk %}'
-   };
+    window.contextVars = window.contextVars || {};
+    window.contextVars.currentUser = {
+        id: -1,
+        fullname: 'Prereg Admin'
+    };
+    window.contextVars.draft = {{ draft | jsonify | safe }};
+    window.contextVars.urls = {
+        updateDraft: '{% url 'pre_reg:comment' draft.pk %}'
+    };
+    // Grab csrf_token
+    function getCookie(name) {
```

Comment:
This is fine, but there are packages for this. We use "jquery.cookie" in the OSF.